.npmrc

```diff
@@ -1 +0,0 @@
-package-lock=false
```

Comment:
There is some value with not having `package-lock.json` in libraries: the tests always run with the latest dependencies version, which is what users of the library are more likely to have.

I usually still store lockfiles, but since this repo currently doesn't have it it's not an "obvious" decision and it could probably be moved to a separate PR.